src/openexr.imageio/exroutput.cpp

```diff
@@ -1729,15 +1729,15 @@ OpenEXROutput::write_deep_scanlines(int ybegin, int yend, int /*z*/,
         frameBuffer.insertSampleCountSlice(countslice);
         std::vector<void*> pointerbuf;
         dd->get_pointers(pointerbuf);
+        size_t slchans      = size_t(m_spec.width) * size_t(nchans);
```

Comment:
The only thing that caused me to do a double-take in this PR is that this line doesn't mirror the form used in exrinput. Can we declare nchans as size_t above (assuming m_spec.nchannels is already known to be >= 0) so that this entire block of code will match what is used in exrinput? i.e. there's 3 casts to size_t in this block whereas the ones in exrinput has none.

Reply:
Yeah, that makes sense

Reply:
So updated (delay was for fixing the great GHA failure of 2024).